### DISCUSSION TITLE
[Support] 在生产环境下不生成log日志的问题

### DISCUSSION THREAD

**Discussion by IceSakuraAzure** [midwayjs/midway#3857] [Q&A] [answered]
### Describe the problem（描述问题）

我通过`npm init midway@latest -y`创建了一个新的简单应用，但是测试的时候发现生产环境下没有输出日志。看了下日志的章节，但是即使把config.prod.ts中的midwayLogger级别配置成了info，也仍然没有看到像dev模式一样生成了访问接口的日志。 开发环境是mac，也看了/home目录下，并没有见到日志文件。所以想问一下，是我个人配置失误，还是存在bug？
注：我并没有在代码中手动用ctx的方式输出log，只是希望像dev一样，记录基本的调用记录即可。
```
//config.prod.ts
import { MidwayConfig } from '@midwayjs/core';

export default {
  // use for cookie sign key, should change to your own and keep security
  keys: '1715741549626_7909',
  koa: {
    port: 7001,
  },
  midwayLogger: {
    level:"info",
  },
} as MidwayConfig;
```

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.16.0)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.16.0)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache-manager(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.16.0)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(not installed)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(3.16.1)
✓ @midwayjs/info(3.16.1)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.16.0)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/mqtt(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(not installed)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.16.0)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(not installed)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/tenant(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(3.16.1)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.16.1)
✓ @midwayjs/ws(not installed)
**********************************************************************
>> Check complete, all versions are healthy.
**********************************************************************

**Comment by waitingsong** [comment]
生产环境下日志路径是 `~/logs/` 你那儿没有么？

**Reply by czy88840616** [reply]
看一下 ~ 的目录是哪个，是跟着你当前的用户走的。

**Reply by IceSakuraAzure** [reply]
目录是生成了的，包括我用自定义方式定义log目录也没有问题，但是没有log文件。。。

**Reply by czy88840616** [reply]
打个error看看

**Reply by IceSakuraAzure** [reply]
我把 level 改成 debug 都不行，四个等级我都试过了

**Reply by czy88840616** [reply]
怎么改的，怎么打的

**Reply by IceSakuraAzure** [reply]
额，大佬我想确认一个事情，midwayjs的生产环境日志是不是哪怕调整成info级别，也不像本地dev的时候 会默认对所有调用自动记录，而是需要自己手动通过logger主动打日志？如果是这样的话，是我误解了日志功能。

**Reply by IceSakuraAzure** [reply]
好像也不是 我看到默认项目report中间键是有默认打了个上下文的info日志的，我直接上传代码吧，我的仓库地址是https://github.com/IceSakuraAzure/midwayTest.git

**Reply by IceSakuraAzure** [reply, marked as answer]
找到原因了，mac下，日志目录不在home/user下，是在/Users/用户/logs/下，我下意识当成linux目录去/home找日志了，打搅了。。。

**Comment by IceSakuraAzure** [comment]
> 怎么改的，怎么打的



> 怎么改的，怎么打的



> 怎么改的，怎么打的
